tools/inflector/inflector.go

```diff
@@ -6,18 +6,17 @@ import (
 	"unicode"
 )
 
-var columnifyRemoveRegex = regexp.MustCompile(`[^\w\.\*\-\_\@\#]+`)
-var snakecaseSplitRegex = regexp.MustCompile(`[\W_]+`)
+var (
+	columnifyRemoveRegex = regexp.MustCompile(`[^\w.*\-_@#]+`)
+	snakecaseSplitRegex  = regexp.MustCompile(`[\W_]+`)
+)
 
 // UcFirst converts the first character of a string into uppercase.
 func UcFirst(str string) string {
-	if str == "" {
-		return ""
+	for i, c := range str {
```

Comment:
This is my opinion makes it harder to read and a little confusing at first. Let's keep the old implementation.